<launch>
    <!--<include file="$(find ptgrey_reader)/launch/single.launch"/>-->
    <!--<node name="cv_ad" pkg="rm_cv" type="armor_detection_node" cwd="ROS_HOME" launch-prefix="sudo -E bash -c" output="screen">-->
    <!--</node>-->
    <node name="gimbal_controller" pkg="gimbal_controller" type="gimbal_controller_node" output="screen">
        <param name="lp_yaw_alpha" type="double" value="0.9"/>
        <param name="lp_pit_alpha" type="double" value="0.9"/>
        <param name="camera_twist" type="string" value="/beckThing"/>
        <param name="publisher_topic" type="string" value="/cmd_vel"/>
    </node>
</launch>
